docs/references/astro/session-list-store.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: $sessionListStore
+description: Clerk's $sessionList store retrieves a list of sessions that have been registered on the client device.
+---
+
+# `$sessionListStore`
+
+The `$sessionListStore` store returns an array of [`Session`](/docs/references/javascript/session) objects that have been registered on the client device.
+
+## How to use the `$sessionListStore` store
+
+The following example demonstrates how to use the `$sessionListStore` store to retrieve a list of sessions that have been registered on the client device. The `isLoaded` property is used to handle the loading state, and the `sessions` property is used to display the number of times the user has visited the page.
```

Comment:
* `isLoaded` isn't correct here.
* I think the code example isn't super clear -- if the user visited the page with the same session 10 times the `length` would still be 1. Maybe we could so something with `sessions[0].expireAt`?

Reply:
Maybe a simple "Your current session expires at `{sessions[0].expireAt}` would do?

Reply:
Yeah, for sure.

Reply:
Updated!